src/java/htsjdk/tribble/dbsnp/OldDbSNPCodec.java

```diff
@@ -89,6 +89,11 @@ public OldDbSNPFeature decode(String line) {
     }
 
     @Override
+    public boolean canDecode(String path){
+	return path.toLowerCase().endsWith(".dbsnp") || path.toLowerCase().endsWith(".rod") || path.toLowerCase().endsWith(".snp");
```

Comment:
Do we really want ".rod" and ".snp" as recognized file extensions for this codec? They are way too generic and non-standard. Just use ".dbsnp"